spmd/tensor/api.py

```diff
@@ -66,6 +67,22 @@ def __repr__(self):
         # TODO: consider all_gather the local tensors for better debugging
         return f"DistributedTensor({self._local_tensor}, placements={self._placements})"
 
+    @classmethod
```

Comment:
Somehow after I rebased on top of https://github.com/pytorch/PiPPy/pull/328, it shows up here. Maybe during merge, we can get the resolved by solving the merge conflict.